### ISSUE TITLE
Use X logo

### ISSUE THREAD

**Issue by Secret-chest** [twitter/opensource-website#184]
Twitter is now X.

**Comment by HelenJonathan** [comment]
Hello @Secret-chest can i handle this ?

**Comment by Secret-chest** [comment]
I'm not interested in contributing so yes.

**Comment by psjnek** [comment]
> Hello @Secret-chest can i handle this ?

I don't think that submitting a PR to this project is worth it as it is inactive. The last change was made a year ago and Elon is probably creating a new GitHub organization anyway.

**Comment by Sayyidalijufri** [comment]
Please merge my pull
